pkg/controllers/cassandra/testing/testing.go

```diff
@@ -174,6 +213,78 @@ func (f *Fixture) AssertServicesLength(l int) {
 	}
 }
 
+func (f *Fixture) ServiceAccounts() *v1.ServiceAccountList {
+	serviceAccounts, err := f.k8sClient.
+		CoreV1().
+		ServiceAccounts(f.Cluster.Namespace).
+		List(metav1.ListOptions{})
+	if err != nil {
+		f.t.Fatal(err)
+	}
+	return serviceAccounts
+}
+
+func (f *Fixture) AssertServiceAccountsLength(l int) {
+	serviceAccounts := f.ServiceAccounts()
+	serviceAccountsLength := len(serviceAccounts.Items)
+	if serviceAccountsLength != l {
+		f.t.Log(serviceAccounts)
+		f.t.Errorf(
+			"Incorrect number of services accounts. Expected %d. Got %d.",
+			l,
+			serviceAccountsLength,
+		)
+	}
+}
+
+func (f *Fixture) Roles() *rbacv1.RoleList {
+	roles, err := f.k8sClient.
+		RbacV1beta1().
+		Roles(f.Cluster.Namespace).
+		List(metav1.ListOptions{})
+	if err != nil {
+		f.t.Fatal(err)
+	}
+	return roles
+}
+
+func (f *Fixture) AssertRolesLength(l int) {
+	roles := f.Roles()
+	rolesLength := len(roles.Items)
+	if rolesLength != l {
+		f.t.Log(roles)
+		f.t.Errorf(
+			"Incorrect number of roles. Expected %d. Got %d.",
+			l,
+			rolesLength,
+		)
+	}
+}
+
+func (f *Fixture) RoleBindings() *rbacv1.RoleBindingList {
+	roleBindings, err := f.k8sClient.
+		RbacV1beta1().
+		RoleBindings(f.Cluster.Namespace).
+		List(metav1.ListOptions{})
+	if err != nil {
+		f.t.Fatal(err)
+	}
+	return roleBindings
+}
```

Comment:
For a future PR - but it'd be great if we can make these sorts of things generic between controllers.

If we define a `RoleBindingsFixture` struct, and embed it within the `Fixture` struct for each controller we can plug and play different test fixture pieces potentially.

Something to think about 😄